pykokkos/lib/ufuncs.py

```diff
@@ -1381,6 +1389,270 @@ def exp2(view):
     return out
 
 
+@pk.workunit
+def sum_impl_1d_double(tid: int, acc: pk.Acc[pk.double], viewA: pk.View1D[pk.double]):
+    acc += viewA[tid]
+
+
+def sum(viewA, axis=None):
+    if(axis is not None):
+        res = np.sum(viewA, axis=axis)
+        view = pk.View(res.shape, pk.double)
+        view[:] = res
+
+        return view
+
+    if str(viewA.dtype) == "DataType.double":
+        return pk.parallel_reduce(
+            viewA.shape[0],
+            sum_impl_1d_double,
+            viewA=viewA)
+
+
+def argmax(view, axis):
+    res = np.argmax(view, axis=axis)
```

Comment:
This will only work if `view` is a numpy array, right? I think the assumption we're making with all these ufuncs is that all array inputs are Views. I understand that for now we need to use numpy, but even if `view` was of type View, you could still access it as a numpy array using the `data` field (see [here](https://github.com/kokkos/pykokkos/blob/a39282171630803067c8c2f13858e0a9beae2eb9/pykokkos/interface/views.py#L282)). So this would be `view.data` instead of just `view`.

Reply:
The type of view in this function is `pk.View2D[pk.double]`. The np.argmax function i think takes as input anything array-like so it is working out of the box thats why. Do you think i should still change it to view.data?

Reply:
If it's working right now it's fine, since we're going to be replacing this eventually